flarefly/fitter.py

```diff
@@ -600,20 +618,21 @@ def plot_mass_fit(self, **kwargs):
         style = kwargs.get('style', 'LHCb2')
         logy = kwargs.get('logy', False)
         figsize = kwargs.get('figsize', (7, 7))
-        bins = kwargs.get('bins', 100)
+        #bins = kwargs.get('bins', 100)
+        bins = self._data_handler_.get_nbins()
         axis_title = kwargs.get('axis_title', self._data_handler_.get_var_name())
+        show_extra_info = kwargs.get('show_extra_info', True)
```

Comment:
Let's put it false not to have crowded plots by default 😉 
```suggestion
        show_extra_info = kwargs.get('show_extra_info', False)
```